### PR TITLE
Bento 2296 Finished login component refactoring

### PR DESCRIPTION
**Pull request by Senait7-ux**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: packages/bento-frontend/public/injectEnv.js [hunk 1]
```diff
@@ -15,8 +15,8 @@ window.injectedEnv = {
   // IDP Auth url
   REACT_APP_NIH_AUTH_URL: 'https://stsstg.nih.gov/auth/oauth/v2/authorize',
   // Client IDs for IDP
-  REACT_APP_GOOGLE_CLIENT_ID: 'Sample id',
-  REACT_APP_NIH_CLIENT_ID: 'Sample id',
+  REACT_APP_GOOGLE_CLIENT_ID: '196014713877-0d926jpdd691roubuc0kpu6r6ha9b9t5.apps.googleusercontent.com',
```
Comment (iksheth) [inline packages/bento-frontend/public/injectEnv.js:18]: Can you remove client Ids values? and replace it with 'Sample id'? We do usually do not commit client id values in github.

**Review by iksheth — CHANGES_REQUESTED** [review]

**Comment by Senait7-ux** [comment]
Bento 2296 PR LINK : https://github.com/CBIIT/bento-frontend/pull/754

File: packages/authentication/src/data/loginProvidersData.js
Comment (iksheth) [inline packages/authentication/src/data/loginProvidersData.js:1]: @Senait7-ux This is custodian configuration file. This should be under "bento-frontend/src/bento" folder.

File: packages/authentication/src/utils/env.js
Comment (iksheth) [inline packages/authentication/src/utils/env.js:1]: @Senait7-ux If I remember correct. This file is part of whole application. It's not part of only login functionality.

File: packages/authentication/src/utils/localStorage.js
Comment (iksheth) [inline packages/authentication/src/utils/localStorage.js:1]: Store to local storage is also used by Cart. This file is not specific to this component.

**Review by iksheth — CHANGES_REQUESTED** [review]

File: packages/authentication/src/data/loginProvidersData.js
Comment (Senait7-ux) [reply packages/authentication/src/data/loginProvidersData.js:1]: Over all i was thinking to have all related functions as well as data and components in a single package folder

File: packages/authentication/src/data/graphQLQueries.js
[the remark this answers is not in the document]
Comment (Senait7-ux) [reply packages/authentication/src/data/graphQLQueries.js:1]: the reason that I have named it graphQLQueries is maybe if other authentication related query to be included and exported in this specific package.

File: packages/authentication/src/utils/env.js
Comment (Senait7-ux) [reply packages/authentication/src/utils/env.js:1]: I did not want to import the env from outside the shared  package folders based on my understanding of the component guideline documentation on Sharepoint.. Is it allowed to do so ?

documemtation: https://nih.sharepoint.com/:w:/r/sites/NCI-CBIIT-FNL-Bento/_layouts/15/Doc.aspx?sourcedoc=%7BB2AB7E52-3989-46B4-8F90-4B88A3BFBE7D%7D&file=Bento%20Component%20Guidelines.docx&action=default&mobileredirect=true

File: packages/authentication/src/utils/getRedirectPath.js
[the remark this answers is not in the document]
Comment (Senait7-ux) [reply packages/authentication/src/utils/getRedirectPath.js:1]: the redirect is based on the query of redirect parameter of the provider authentication url

File: packages/bento-frontend/src/components/Auth/AuthProvider.js
[the remark this answers is not in the document]
Comment (Senait7-ux) [reply packages/bento-frontend/src/components/Auth/AuthProvider.js:1]: No it has not been removed, I just moved the signIn with google and NIH to the package and was planning to remove authProvider completely from the auth to the authentication package including the login component in the header.

**Comment by iksheth** [comment]
Closing this PR as a new PR #778 has been created regarding the same changes with new structure.

**Pull request closed without merging**